examples/inference-deployments/instructor/instructor_handler.py

```diff
@@ -18,16 +18,16 @@ def __init__(self, model_name: str):
         self.model = INSTRUCTOR(self.model_name)
         self.model.to(self.device)
 
-    def predict(self, inputs: List[Dict[str, Any]]):
+    def predict(self, model_requests: List[Dict[str, Any]]):
```

Comment:
don't need to change anything, but: now that we're using a nested dict here, might be worth simplifying these annotations (`Dict` instead of `Dict[str, Any]`).